exasol/slc/internal/tasks/test/test_runner_db_test_base_task.py

```diff
@@ -239,8 +245,9 @@ def get_result_status(status):
 
     def get_test_folders(self, test_config):
         test_folders = []
-        if test_config["test_folders"] != "":
-            test_folders = test_config["test_folders"].split(" ")
+        for test_set in test_config.test_sets:
+            for test_folder in test_set.folders:
+                test_folders.append(test_folder)
```

Comment:
I'am thinking now, if we should filter the test sets per release goal and accelerator.
@tkilias what do you think?

Reply:
The release goal information we have in the CI and also in Luigi task and the tests only make sense for these so this is a good idea .

How would we do that for the accelerator?

Reply:
The accelerator is already passed as argument to run-db-test (it's needed to spawn the ITDE with the apropriate parameters). However, I believe it's not yet passed to the Luigi Tasks....